Lesson24/selcan1/1.html

```diff
@@ -0,0 +1,127 @@
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         .sagEkran{
+            display: flex;
+            margin-left: 90px;
+            justify-content: center;
+         }
+        .sagEkran>img{
+            width: 90%;
+            height: 100%;
+            border-radius:20px;
+        }
+         h2{
+            font-size: 30px;
+            padding: 0 20px;
+         }
+         ul{
+            line-height: 30px;
+            padding-left: 0;
+         }
+         .solEkran li{
+            list-style-type: none;
+            padding-left:10px;  
+         }
+         ul .kutu{
+            display: flex;
+            align-items: flex-start;
+         }
+         .form-input{
+            width: 100%;
+            height: 35px;
+            border-radius: 5px;
+            margin-bottom: 20px;
+         }
+         .form-input,.form-label{
+            letter-spacing: 2px;
+            margin-top: 10px;
+         }
+         button{
+            width:calc(100% + 10px);
+            height: 50px;
+            background-color: rgb(4, 4, 30);
+            color: white;
+            padding: 5px auto;
+            border-radius: 10px;
+            cursor: pointer; 
+         }
+         button:hover{
+            background-color: #590808;
+            transition: .5s;
+         }
+         .error {
+            color: #590808;
+            font-size: 12px;
+            margin-top: 5px;
+            margin-bottom: 10px;
+        }
+
+        
+
+
+    </style>
+</head>
+<body>
+    <div class="kapsayici">
+        
+       <div class="anaMenu">
+        <div class="solEkran">
+            <h2>Stay Updated!</h2>
+            <p>Join 60.0004 product managers receiving monthly updates on:</p>
+           <ul>
+               <div class="kutu"> <input type="checkbox"><li>Product discovery and building what matters</li></div>  
+                <div class="kutu"><input type="checkbox"><li>Measuring to ensure updates are success </li></div>
+                <div class="kutu"><input type="checkbox"><li>And much more!</li></div>
+            </ul>
+            <form id="subscribeForm" novalidate>
+            <label for="email" class="form-label" >Email Address:</label>
+            <div id="emailError" class="error"></div>
+            <input type="email" id="email" name="email" class="form-input" placeholder="Email@company.com"required>
+            <div class="buton"><button>Subscribe to monthly newsletter</button></div></form>
+        </div>
+        <div class="sagEkran">
+            <img src="./1.png" alt="resim1" ">
+        </div>
+        </div>
+    
+    </div>
+    <script>
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+        
+        const form = document.getElementById("subscribeForm");
+        form.addEventListener("button",function(event){
```

Comment:
You want to do validation when the form is submitted, the event listener type is "submit" in this case. 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event

Reply:
İt is solved with such a little change:)Thanks...
We will always use "submit" in such forms then,true?

Reply:
When you need to add an event listener to a form element, it's usually "submit" type event